src/components/ShoppingList/PopulatedList.js

```diff
@@ -36,7 +37,20 @@ const PopulatedList = () => {
   };
 
   const deleteItemHandler = (id) => {
-    db.collection(localStorage.getItem("token")).doc(id).delete();
+    swal({
+      title: "Are you sure?",
+      text: "Once deleted, you cannot revert back.",
+      icon: "warning",
+      buttons: true,
+      dangerMode: true,
+    }).then((willDelete) => {
+      if (willDelete) {
+        db.collection(localStorage.getItem("token")).doc(id).delete();
+        swal("Item deleted!", {
+          icon: "success",
+        });
+      }
+    });
```

Comment:
This modal package is awesome! Nice find. Way to implement it all. It's got me excited to start styling.